03.bowling/bowling.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = scores.map { |s| s == 'X' ? 10 : s.to_i }
+frame_idx = 0
+first_done = false
+LAST_FLAME = 10
+result = Array.new(LAST_FLAME, 0)
```

Comment:
```suggestion
total = 0
```

配列にしなくても数値の変数を用意してそこに得点を足し込んでいく形でもロジックはほとんど変えなくて済みそうです。
（ただし、20行目の条件分岐だけは一工夫いりますが）